### ISSUE TITLE
create tag albums in a album

### ISSUE THREAD

**Issue by pchampio** [LycheeOrg/Lychee#1567]
Before lychee v4.6.0 a tag album could be added (moved actually) to any album.
This was super useful to create restricted version of an album by only tagging the images and keeping the album structure nice and organized.
Here is an example:

```log
/
├─ vacances mer/
│  │  ├─ photo_1_real (tag fav_vm)
│  │  ├─ photo_2_real
│  │  ├─ photo_3_real
│  │  ├─ [....]
│  ├─ favorites/ (tag fav_vm)
│  │  ├─ photo_1_tag
├─ vacances soleil/
│  │  ├─ photo_1_real (tag fav_vs)
│  │  ├─ photo_2_real
│  │  ├─ photo_3_real (tag fav_vs)
│  │  ├─ [....]
│  ├─ favorites/ (tag fav_vs)
│  │  ├─ photo_1_tag
│  │  ├─ photo_3_tag
```

The lookup of tagged images works globally (not per albums), it is up to the user to set different 'fav' identifier like in the example.

**Comment by kamil4** [comment]
Yeah, I see what you were doing there, but this was never meant to be a supported scenario. Tag albums as currently implemented were meant to reside at the top level only and being able to move them to another album was basically a bug, which  we've fixed now.

There is of course a larger conversation to be had about how best to support a scenario like the one you outlined above; in fact, we've had at least some of that conversation already, though without reaching any conclusions (or at least without anybody implementing any replacement so far).

Let's make sure though that we fully understand how you _want_ this to work. You would like to keep all the photos from a particular event in a single album, but then you would like to make a _subset_ of these photos public, correct? And you want to avoid having the public vs non-public photos in separate albums (which is how this could be solved now)? Or are all the photos (and their containing album) public and you simply want to have an additional grouping of favorite ones (and you don't want to simply use the current "Copy" functionality, presumably because the metadata can then get of sync between the copies?).

**Comment by pchampio** [comment]
> You would like to keep all the photos from a particular event in a single album, but then you would like to make a subset of these

yes.

> subset of these photos public, correct?

yes, but In my case, the 'non-favorite' album is also public. (we will come to this later).

> And you want to avoid having the public vs non-public photos in separate albums (which is how this could be solved now)?

Everything is public in my case.

> 
> [.....] (which is how this could be solved now) ?
>  Or are all the photos (and their containing album) public and you simply want to have an additional grouping of favorite ones (and you don't want to simply use the current "Copy" functionality, presumably because the metadata can then get of sync between the copies?)

Actually, even if the tag album is shared, it cannot 'unlock' the photos it contains due to this ownership issue. 

I have missed the "Copy" functionality. I should be using it!
It solves the ownership issue, and I understand the use case now. 

I quite like tag albums. As you said, the metadata can then sync between the copies and keeps the database nicely organized.
But it is just not the right tool for what I want to do. And I also don't see how to solve the ownership issue without having to rely on a form of copy.

Thanks for your comment and good questions that made me realize the tag album limitation and how 'Copy' is not that bad, actually :heart: !
You can close this issue.
